homeassistant/helpers/device_registry.py

```diff
@@ -139,6 +139,14 @@ def async_update_device(
         return self._async_update_device(
             device_id, area_id=area_id, name_by_user=name_by_user)
 
+    @callback
+    def async_migrate_device_identifiers(self, device_id, identifiers: set):
```

Comment:
Can we make this part of update device?

Reply:
Yes - with the addition of a `new_identifiers` parameter.